components/contract-components/contract-release-form/landing-fieldset.tsx

```diff
@@ -0,0 +1,303 @@
+          <FormControl
+            isInvalid={!!form.formState.errors.displayName}
+            isRequired
+          >
+            <FormLabel>Release Name</FormLabel>
+            <Input {...form.register("displayName")} required />
+            <FormErrorMessage>
+              {form.formState.errors?.displayName?.message}
+            </FormErrorMessage>
+          </FormControl>
+          <FormControl isInvalid={!!form.formState.errors.description}>
+            <FormLabel>Description</FormLabel>
+            <Textarea {...form.register("description")} rows={2} />
+
+            <FormErrorMessage>
+              {form.formState.errors?.description?.message}
+            </FormErrorMessage>
+          </FormControl>
+        </Flex>
+      </Flex>
+
+      <Box>
+        <Heading size="title.md" mb={2}>
+          README
+        </Heading>
+        <Text size="body.md" mb={4}>
+          Describe what your contract does and how it should be used. Markdown
+          formatting is supported.
+        </Text>
+        <FormControl isInvalid={!!form.formState.errors.readme}>
+          <Tabs isLazy lazyBehavior="keepMounted" colorScheme="purple">
+            <TabList
+              px={0}
+              borderBottomColor="borderColor"
+              borderBottomWidth="1px"
+            >
+              <Tab gap={2}>
+                <Icon as={BsCode} my={2} />
+                <Heading size="label.lg">Write</Heading>
+              </Tab>
+              <Tab gap={2}>
+                <Icon as={BsEye} my={2} />
+                <Heading size="label.lg">Preview</Heading>
+              </Tab>
+            </TabList>
+            <TabPanels pt={2}>
+              <TabPanel px={0} pb={0}>
+                <Textarea {...form.register("readme")} rows={12} />
+                <FormErrorMessage>
+                  {form.formState.errors?.readme?.message}
+                </FormErrorMessage>
+              </TabPanel>
+              <TabPanel px={0} pb={0}>
+                <Card>
+                  <MarkdownRenderer markdownText={form.watch("readme") || ""} />
+                </Card>
+              </TabPanel>
+            </TabPanels>
+          </Tabs>
+        </FormControl>
+      </Box>
+      <Box>
+        <Heading size="title.md" mb={2}>
+          Version information
+        </Heading>
+        <Text size="body.md" mb={4}>
+          Set your contract version number, add release notes, and link to your
+          contract&apos;s audit report.
+        </Text>
+        <Flex flexDir="column" gap={6}>
+          <FormControl isRequired isInvalid={!!form.formState.errors.version}>
+            <Flex alignItems="center" mb={1}>
+              <FormLabel flex="1" mb={0}>
+                Version
+              </FormLabel>
+              {latestVersion && (
+                <Text size="body.md">latest release: {latestVersion}</Text>
+              )}
+            </Flex>
+            <Input
+              {...form.register("version")}
```

Comment:
we should be using `register('version', { required: true })`

Reply:
fixed in https://github.com/thirdweb-dev/dashboard/pull/782/commits/4085423a51450a30723a9dad73401a21d3fdd846